src/test/scala/mesosphere/marathon/test/MarathonTestHelper.scala

```diff
@@ -392,6 +393,23 @@ object MarathonTestHelper {
     None
   )
 
+  def makeCreatedInstance(
```

Comment:
Since there is a PR removing `Created` from the conditions - do you think it makes sense to change this to `Provisioned`?

Reply:
Yeah, I'll change it.